src/doc/python_scripting/functions.rst

```diff
@@ -12175,15 +12178,42 @@ WriteConfigFile
 WriteScript
 -----------
 
+**Synopsis:**
+
+::
+
+  WriteScript(f)
```

Comment:
Where can I find the return type for this function? I feel like I looked everywhere.

Reply:
It returns Nonetype